README.md

```diff
@@ -20,3 +20,30 @@ The realtime documentation can be found [here](http://docs.beeswax.com/docs/abou
       <version>2019-04-02</version>
     </dependency>
 
+
+# Tools
+
+### Augmentor requests generator
+
+    beeswax/tools/augmentor/
+
+It generates Beeswax augmentor HTTP requests and sends them to a designated endpoint.
```

Comment:
Can you remove the "It" from each of these bullets?

Reply:
Removed